dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.internal.optional.jaxb-2.3.feature

```diff
@@ -10,7 +10,7 @@ Subsystem-Name: Java XML Bindings 2.3 for all Java versions
 -features=\
   com.ibm.websphere.appserver.classloading-1.0
 -bundles=\
-  com.ibm.websphere.javaee.activation.1.1; location:="dev/api/spec/,lib/"; apiJar=false, \
+  com.ibm.websphere.javaee.activation.1.1; require-java:="9" ;location:="dev/api/spec/,lib/"; apiJar=false, \
```

Comment:
I thought for 2.3 we required activation 1.1 independent of java 9 or not because jaxb 2.3 requires activation 1.1.

Reply:
Going to leave this for now to avoid different versions of the same Activation spec being available when running on JDK 8